.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+# Reality Collective - Reality Toolkit Pull Request

```

Comment:
Should we add testing instruction area?

Reply:
Good idea. We should revisit this template in general